### PR TITLE
Replace usages of deprecated accompanist-systemuicontroller

### PR DESCRIPTION
**Pull request by Tommy-Geenexus**

### CODE REVIEW

File: app/src/main/java/app/accrescent/client/ui/MainActivity.kt [hunk 1]
```diff
@@ -109,21 +97,27 @@ class MainActivity : ComponentActivity() {
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun MainContent(
-    systemUiController: SystemUiController,
-    appId: String?
+    appId: String?,
+    modifier: Modifier = Modifier
```
Comment (lberrymage) [inline app/src/main/java/app/accrescent/client/ui/MainActivity.kt]: nit: Could you move the modifier parameter addition to a separate commit and update the lint baseline in that commit to remove this instance?

**Review by lberrymage — CHANGES_REQUESTED** [review]
Sweet, thank you!

File: app/src/main/java/app/accrescent/client/ui/MainActivity.kt (hunk 1 above)
Comment (Tommy-Geenexus) [reply app/src/main/java/app/accrescent/client/ui/MainActivity.kt]: Addition has been reverted, better fit for more in depth linting.

File: gradle/verification-metadata.xml
Comment (lberrymage) [inline gradle/verification-metadata.xml:1]: Great. Once the merge conflict here is resolved this looks good to go 👍🏻.

Comment (Tommy-Geenexus) [reply gradle/verification-metadata.xml:1]: Forgot rebase, should be good now.

**Review by lberrymage — APPROVED** [review]

**Comment by lberrymage** [comment]
Looks good, thank you!
